### PR TITLE
Documentation refresh

### PR DESCRIPTION
**Pull request by emilymdubois**
This PR pulls documentation usage details out of the main `readme.md` and into specific files in the `docs/` directory.

cc @rclark @emilymcafee

### CODE REVIEW

File: docs/debugging-failures.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Runtime resources:
+Cluster ARN: arn:aws:ecs:us-east-1:123456789012:cluster/ecs-cluster-production
+Instance ARN: arn:aws:ecs:us-east-1:123456789012:container-instance/2e14b317-0909-4ecc-ab88-d94fe64d2167
+Task ARN: arn:aws:ecs:us-east-1:123456789012:task/798b49eb-49d7-4abb-a305-82a6e723caf6
+```
+
+First off all, check the `Message` JSON to help identify the commit that caused a failure, the repository that was being built, and the person who was responsible for the commit.
+
+Next, use the `MessageId` (`a7492004-8ca8-4322-9299-2e82bb649163` in this example) to search container logs. Logs from ecs-conex containers will be written to `/var/log/messages` on the host EC2s (assuming you're running ecs-conex on a EC2s started from [ECS-optimized AMIs](http://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs-optimized_AMI.html)). If you run on an ECS cluster with more than one EC2, you may have to use the `Instance ARN` in an `ecs:DescribeContainerInstances` request to determine the EC2 that the container ran on.
```
Comment (rclark) [inline docs/debugging-failures.md]: The statements here and in logging.md about `/var/log/messages` are no longer valid. [The LogConfiguration in ecs-watchbot](https://github.com/mapbox/ecs-watchbot/blob/master/lib/template.js#L407-L414) now forwards conex worker logs to a cloudwatch log group.

The LogGroup's name is defined here: https://github.com/mapbox/ecs-watchbot/blob/master/lib/template.js#L165, and this is where you'd want to look for logs / debuggery.

**Comment by emilymdubois** [comment]
@rclark, I replaced the EC2 logging language with information about ecs-watchbot logging aggregation.

How do you feel about linking back to ecs-watchbot pages? I linked to `lib/template.js`, but avoided any line references in case those change significantly. This may not be necessary; pointing to ecs-watchbot documentation may be enough.

**Comment by rclark** [comment]
I'd say leave out the links to watchbot code in favor of just stating what the log group name is going to look like.

**Comment by emilymdubois** [comment]
@rclark I think this is ready to merge if you're 👍 !

**Comment by rclark** [comment]
:+1: thanks!

**Pull request merged**